test/higher_order_ops/test_with_effects.py

```diff
@@ -371,6 +373,80 @@ def add_hooks(module, config):
             self.assertTrue("MockModule.linear:mean" in recorded_dict)
             self.assertTrue("MockModule:mean" in recorded_dict)
 
+    @skipIfNoDynamoSupport
+    def test_effectful_custom_op_with_subclass(self):
+        with torch.library._scoped_library("_mylib", "FRAGMENT") as lib:
+            lib.define("foo(Tensor x) -> Tensor")
+
+            def foo_impl(a):
+                torch.ops.aten._print("fwd")
+                return a.clone()
+
+            def foo_bwd(ctx, grad):
+                torch.ops.aten._print("bwd")
+                return grad.clone()
```

Comment:
instead of printing, you could have these tick a counter on a global variable. Then, we can check that the global variables were mutated the correct amount in these tests.